views/settings/partials/performance-monitoring/global-score-widget.php

```diff
@@ -5,47 +5,49 @@
 
 defined( 'ABSPATH' ) || exit;
 ?>
-<div class="wpr-optionHeader">
-	<h3 class="wpr-title2">
-		<?php echo esc_html__( 'RocketInsights', 'rocket' ); ?>
-	</h3>
-</div>
-<div class="wpr-fieldsContainer">
-	<fieldset class="wpr-fieldsContainer-fieldset">
-		<div class="wpr-field">
-			<div class="wpr-percentage-score-widget">
-				<div>
+<div id="wpr_global_score_widget">
+	<div class="wpr-optionHeader">
+		<h3 class="wpr-title2">
+			<?php echo esc_html__( 'RocketInsights', 'rocket' ); ?>
+		</h3>
+	</div>
+	<div class="wpr-fieldsContainer">
+		<fieldset class="wpr-fieldsContainer-fieldset">
+			<div class="wpr-field">
+				<div class="wpr-percentage-score-widget">
+					<div>
+						<?php
+						if ( isset( $data['status'] ) && 'no-url' !== $data['status'] ) :
+							$this->render_performance_score( $data );
+							?>
+						<?php else : ?>
+							<div class="wpr-score-no-urls"></div>
+						<?php endif; ?>
+					</div>
+					<p class="wpr-page-num-txt">
+					<?php
+					// translators: %1$s is the number of pages monitored.
+					printf( esc_html__( '%1$s page monitored.', 'rocket' ), intval( $data['pages_num'] ) );
+					?>
+					</p>
 					<?php
-					if ( isset( $data['status'] ) && 'no-url' !== $data['status'] ) :
-						$this->render_performance_score( $data );
-						?>
-					<?php else : ?>
-						<div class="wpr-score-no-urls"></div>
-					<?php endif; ?>
+					$this->render_action_button(
+						'link',
+						'',
+						[
+							'label'      => $data['pages_num'] ? __( 'ADD PAGE', 'rocket' ) : __( 'ADD HOME PAGE', 'rocket' ),
```

Comment:
```suggestion
							'label'      => $data['pages_num'] ? __( 'Add Page', 'rocket' ) : __( 'Add Homepage', 'rocket' ),
```

If we want all uppercase, we should handle that in CSS instead in my opinion.

Reply:
This wasn't from my PR, but I'll handle it.